### ISSUE TITLE
Unable to set column to citext

### ISSUE THREAD

**Issue by brandonmp** [hasura/graphql-engine#2812]
Using this docker-compose config, I am unable to set column types to `citext`. Have witnessed the same problem on a previous version (beta.4) using the Heroku button deploy config.

Hoping this is user error & i can fix somehow. 

```
version: '3.0'
services:
  postgres:
    image: postgres:9.6.11-alpine
    container_name: postgres
    restart: always
  graphql-engine:
    image: hasura/graphql-engine:v1.0.0-beta.6.cli-migrations
    container_name: hasura
    ports:
      - '9999:8080'
    depends_on:
      - 'postgres'
    restart: always
    volumes:
      - ./migrations/migrations:/hasura-migrations
    environment:
      HASURA_GRAPHQL_DATABASE_URL: postgres://postgres:@postgres:5432/postgres
      HASURA_GRAPHQL_ENABLE_CONSOLE: 'false'
      HASURA_GRAPHQL_LOG_LEVEL: 'error'
      HASURA_GRAPHQL_ENABLED_LOG_TYPES: startup
```
--
The extension adds fine:
![image](https://user-images.githubusercontent.com/14139991/64088948-95482600-ccf8-11e9-866a-647bfb75b508.png)

--
Then, querying `pg_extensions` confirms the extension is there:

![image](https://user-images.githubusercontent.com/14139991/64088960-a2651500-ccf8-11e9-91c6-393a6ffa1a32.png)

--
However, when trying to alter a column to citext, we get an error:

![image](https://user-images.githubusercontent.com/14139991/64088971-b6107b80-ccf8-11e9-988e-65dd2b2a10c0.png)
--
...the details of which (from the toast) are:
```
{
    "path": "$.args[0].args",
    "error": "query execution failed",
    "internal": {
        "arguments": [],
        "error": {
            "status_code": "42704",
            "exec_status": "FatalError",
            "message": "type \"citext\" does not exist",
            "description": "",
            "hint": ""
        },
        "prepared": false,
        "statement": "ALTER TABLE app.users ALTER COLUMN username TYPE citext;"
    },
    "code": "postgres-error"
}
```
--
... and the network call response of which is: 
```
{
  "code": "data_api_error",
  "message": "{\"path\":\"$.args[0].args\",\"error\":\"query execution failed\",\"internal\":{\"arguments\":[],\"error\":{\"status_code\":\"42704\",\"exec_status\":\"FatalError\",\"message\":\"type \\\"citext\\\" does not exist\",\"description\":\"\",\"hint\":\"\"},\"prepared\":false,\"statement\":\"ALTER TABLE app.users ALTER COLUMN username TYPE citext;\"},\"code\":\"postgres-error\"}"
}
```
--
Lastly, the `citext` value doesn't show up in the dropdown: 
![image](https://user-images.githubusercontent.com/14139991/64089028-04257f00-ccf9-11e9-8fe0-5f673f1ec0b4.png)

**Comment by ecthiender** [comment]
On the console what postgres schema is selected when you're running the `CREATE EXTENSION..` command?

I think the issue is that the extension gets created in the `public` schema and not in `app` schema.

Can you try the following and see if it works?

```sql
CREATE EXTENSION IF NOT EXISTS citext SCHEMA app;
```

**Comment by marionschleifer** [comment]
Hey @brandonmp just checking in if you were able to solve the problem? 🙂

**Comment by brandonmp** [comment]
as is too often the case I can't figure out what precisely was going wrong here, but it's fixed, so closing. 

tl;dr: a postgres thing, not a hasura issue. 

for posterity, @ecthiender , i think i actually unintentionally lied in my OP--i think i originally used the command you shared (`CREATE EXTENSION IF NOT EXISTS citext SCHEMA app;`) to create the extension. 

This is why the value in `pg_extension.extnamespace`  was `16630`. 

When I dropped the extension, then ran the command from my OP (`CREATE EXTENSION citext;`), `pg_extension.extnamespace` was `2200`, and everything worked as intended.
